1209/EBA7/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: USBscope
+owner: Victor Grigoryev
```

Comment:
Finally [index.md](https://github.com/VictorGrig/pidcodes.github.com/blob/VictorGrig-USBscope-PID-pull-request/1209/EBA7/index.md) should be like this, isn`t it?

```markdown
---
layout: pid
title: USBscope
owner: USBscope
license: MIT
site: http://yveslebrac.blogspot.com/2008/10/cheapest-dual-trace-scope-in-galaxy.html
source: https://github.com/VictorGrig/USBscope
---
```